frontend/src/core/comments/components/Comment.css

```diff
@@ -44,3 +45,25 @@
     margin-top: -2px;
     padding-left: 8px;
 }
+
+.comments-list .comment .info .pin-button:before {
+    content: "•";
+    padding: 0 3px 0 3px;
+}
+
+.comments-list .comment .info .pin-button {
+    background-color: transparent;
+    border: none;
+    color: #AAAAAA;
+    font-size: 11px;
+    font-weight: 300;
+    padding: 0;
+}
+
+.comments-list .comment .comment-pin {
+    color: orange;
```

Comment:
I'd change this block a little to make the pin icon smaller and its color same as the pinned comment count:

```css
.comments-list .comment .comment-pin {
    color: #7BC876;
    position: absolute;
    right: 4px;
    top: -6px;
}
```